### PR TITLE
Eduge

### PR DESCRIPTION
**Pull request by enod**
Hi, awesome folks behind the huggingface! 

Here is my PR for the text classification dataset in Mongolian.

Please do let me know in case you have anything to clarify. 

Thanks & Regards,
Enod

### CODE REVIEW

File: datasets/eduge/README.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+{
+ 'label': 'урлаг соёл',
+ 'news': 'Шударга өрсөлдөөн, хэрэглэгчийн төлөө газар 2013 оны дөрөвдүгээр сараас эхлэн Монгол киноны ашиг орлогын мэдээллийг олон нийтэд хүргэж байгаа. Ингэснээр Монголын кино үйлдвэрлэгчид улсад ашиг орлогоо шударгаар төлөх, мөн  чанартай уран бүтээлийн тоо өсөх боломж бүрдэж байгаа юм.',
+}
+```
+
+### Data Fields
+
+- `news`: a complete news article on a specific topic
+- `label`: the single class of the topic
```
Comment (lhoestq) [inline datasets/eduge/README.md:86]:
```suggestion
- `news`: a complete news article on a specific topic as a string
- `label`: the single class of the topic, among these values: "урлаг соёл" (0), "эдийн засаг" (1), "эрүүл мэнд" (2), "хууль" (3), "улс төр" (4), "спорт" (5), "технологи" (6), "боловсрол" (7), "байгал орчин" (8).
```

File: datasets/eduge/README.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+The text in the dataset is in Mongolian
+
+## Dataset Structure
+
+### Data Instances
+
+For the `default` configuration:
+```
+{
+ 'label': 'урлаг соёл',
```
Comment (lhoestq) [inline datasets/eduge/README.md:78]:
```suggestion
 'label': 0, #  'урлаг соёл'
```

File: datasets/eduge/eduge.py [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+                        "урлаг соёл",
+                        "эдийн засаг",
+                        "эрүүл мэнд",
+                        "хууль",
+                        "улс төр",
+                        "спорт",
+                        "технологи",
+                        "боловсрол",
+                        "байгал орчин",
+                    ],
```
Comment (lhoestq) [inline datasets/eduge/eduge.py:77]: There's no need to repeat the labels :)

```suggestion
```

File: datasets/eduge/README.md [hunk 4]
```diff
@@ -0,0 +1,152 @@
+---
```
Comment (lhoestq) [inline datasets/eduge/README.md:1]: Is there a pretty name we can add ?
For example
```suggestion
---
pretty_name: Eduge
```

**Review by lhoestq — COMMENTED** [review]
Good job ! Thanks for adding this dataset :)

I added a few comments

File: datasets/eduge/README.md [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+
+### Licensing Information
+
+[Needs More Information]
+
+### Citation Information
+
+```
+TBD
+```
```
[the remark this answers is not in the document]
Comment (enod) [reply datasets/eduge/README.md:149]: I don't think so.

File: datasets/eduge/README.md [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+- monolingual
+size_categories:
+- unknown
+source_datasets:
+- original
+task_categories:
+- text-classification
+task_ids:
+- multi-class-classification
+paperswithcode_id: eduge
```
[the remark this answers is not in the document]
Comment (enod) [reply datasets/eduge/README.md]: It has been added https://paperswithcode.com/dataset/eduge

[the remark this answers is not in the document]
Comment (lhoestq) [reply datasets/eduge/README.md]: Nice !

**Review by lhoestq — APPROVED** [review]
Looks all good to me !

I just removed the _CITATION variable since it's unnecessary.

Thanks a lot for adding this one :)

**Pull request merged**